src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py

```diff
@@ -30,10 +31,11 @@
 
 class PytorchExportedProgramAdapterImpl:
 
-  def __init__(self, ep: torch.export.ExportedProgram):
+  def __init__(self, ep: torch.export.ExportedProgram, settings: Dict):
```

Comment:
Could you also add the `settings` parameter to the following api too? In its comment, mention that currently we only support 'const_element_count_limit'.

https://github.com/google-ai-edge/model-explorer/blob/main/src/server/package/src/model_explorer/config.py#L68

Reply:
thanks for catching it, Jing!